src/estimators/stratified_sketch.py

```diff
@@ -0,0 +1,294 @@
+# Copyright 2020 The Private Cardinality Estimation Framework Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+import functools
+from wfa_cardinality_estimation_evaluation_framework.estimators.base import SketchBase
+from wfa_cardinality_estimation_evaluation_framework.estimators.base import EstimatorBase
+from wfa_cardinality_estimation_evaluation_framework.estimators.exact_set import ExactMultiSet
+
+ONE_PLUS = '1+'
+
+
+class StratifiedSketch(object):
```

Comment:
How do we expect this class to integrate with the Simulator?

In its current form I think it may be incompatible.

Can you update https://github.com/world-federation-of-advertisers/cardinality_estimation_evaluation_framework/blob/master/examples/basic_comparison.py to include StratifiedSketch?

@shengma-1 @matthewclegg 

Also, it should inherit SketchBase.

Reply:
Updating basic_comparison.py sounds like a nice idea.  For example, we could add a StratifiedSketch of ExactMultiSets as a verification.

Not sure that I agree with the suggestion of inheriting from SketchBase.  SketchBase includes the add(id) method, and there is not an obvious implementation of add(id) for StratifiedSketches.

Reply:
Also, FYI, I think there are at least 2 more PR's in the pipeline that relate to the StratifiedSketch, so this is not the final word.  Everything will integrate in the end.

Reply:
The VoC/ADBF frequency dedupe will this class. See a new PR #44.

Reply:
This is now integrated with the simulator.